### DISCUSSION TITLE
I made a pixelit server based on python

### DISCUSSION THREAD

**Discussion by dassystem** [pixelit-project/PixelIt#284] [Show and tell]
Hi there,

since I did not want to use node red for my pixelit, I have created a python based library and a small server one can install on a raspberry pi to cycle through some pixelit apps. It uses REST or MQTT to send the data.

Feel invited to check it out and leave comments and ideas for improvements.

https://github.com/dassystem/pixelit-server

Thanks for the good work on the pixelit-project. I'm very happy to use this open source software.

**Comment by foorschtbar** [comment]
Nice! Would you like to link your project in the README and/or Docs? I will approve the PR :)

**Reply by dassystem** [reply]
I'm totally fine having my project linked somewhere if it helps somebody. But I cannot decide where the best place for it would be. So feel free to put it somewhere you find appropriate.
